<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiDomainWhois</class>
 <widget class="QWidget" name="UiDomainWhois">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web Domains WHOIS</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Domain:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_DomainName">
     <property name="placeholderText">
      <string>test.com</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_Lookup">
     <property name="text">
      <string>Lookup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit_LookupResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
